src/NuGet.Core/NuGet.Common/project.json

```diff
@@ -22,19 +22,24 @@
         "System.Core": "",

```

Comment:
We don't need "System", "System.Core", and "System.Runtime" right? I thought they are implicit.